### PR TITLE
Pf4 the namespace selector

### PR DESCRIPTION
**Pull request by mtho11**
** Describe the change **
As the title says, this is migrating the namespace selector to PF4.

** Screenshot **

Add a screenshot of the UI after your changes.

### CODE REVIEW

File: src/components/NamespaceDropdown.tsx [hunk 1]
```diff
@@ -3,40 +3,34 @@ import { connect } from 'react-redux';
...
 import Namespace from '../types/Namespace';
-import { PfColors } from './Pf/PfColors';
 import { HistoryManager, URLParam } from '../app/History';
 import {
   BoundingClientAwareComponent,
   PropertyType
 } from './BoundingClientAwareComponent/BoundingClientAwareComponent';
 import { KialiIcon } from 'config/KialiIcon';
-import TourStopContainer from './Tour/TourStop';
-import { GraphTourStops } from 'pages/Graph/GraphHelpTour';
```
Comment (jshaughn) [inline src/components/NamespaceDropdown.tsx]: You need to add back the TourStop for the graph tour.

File: src/components/NamespaceDropdown.tsx [hunk 2]
```diff
@@ -189,21 +219,15 @@ export class NamespaceDropdown extends React.PureComponent<NamespaceDropdownProp
   }
 
   render() {
-    const popover = <Popover id="namespace-list-layers-popover">{this.getPopoverContent()}</Popover>;
+    const { isOpen } = this.state;
     return (
-      <TourStopContainer info={GraphTourStops.Namespaces}>
```
Comment (jshaughn) [inline src/components/NamespaceDropdown.tsx]: This is the TourStop, it needs to wrap the component that roots the Popover.

File: src/components/NamespaceDropdown.tsx (hunk 1 above)
Comment (mtho11) [reply src/components/NamespaceDropdown.tsx]: Ah, right!

File: src/components/NamespaceDropdown.tsx (hunk 2 above)
Comment (jshaughn) [reply src/components/NamespaceDropdown.tsx]: Tour stop is back!

**Comment by aljesusg** [comment]
This fix https://github.com/kiali/kiali/issues/1790

**Pull request merged**